front/app/containers/Admin/pages-menu/constants.ts

```diff
@@ -0,0 +1,7 @@
+import messages from './messages';
```

Comment:
I think just convention over time. I can't speak for the initial decisions but in this case:

`pages-menu` kind of represents a route (`admin/pages-menu`) so it makes sense to have it match the URL route, which is all lowercase
`Admin` is a container, which represents a React component (`<Admin />`) and those are always capitalized when used in React code (this applies to `StickyContainer` too)
`constants` is not a component and just a TS file, so it doesn't need to be capitalized. I imagine it's lowercase because most JS variables are lowercase so it matches that convention